c/cruise-control/redhat_ubi8/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
Why not use a base of ubi8/openjdk-8 here instead of installing it separately?  Should be faster.

Reply:
Correct @gerrith3, but as you might recall I reported two weeks ago that ubi8/openjdk-8 image was not working, it had JBOS as the user and not root.